src/osvr/ClientKit/ServerAutoStartC.cpp

```diff
@@ -38,6 +38,7 @@
 // Standard includes
 // - none
 
+// @todo use a thread-safe lazy-initialized singleton pattern
 #if defined(OSVR_ANDROID)
 static osvr::server::ServerPtr gServer;
```

Comment:
Did you want to do anything about this before merge, or will you file an issue, or is it not expected to be a big deal?